addons/pos_hr/static/src/js/models.js

```diff
@@ -28,7 +34,8 @@ models.load_models([{
     },
 },{
     model:  'hr.employee',
-    fields: ['name', 'id', 'barcode', 'pin', 'user_id'],
+    fields: ['name', 'id', 'barcode', 'pin', 'user_id', 'company_id'],
```

Comment:
company_id is the same as the one in `self.config`, so this field doen't need to be loaded.